server/src/main/java/org/opentosca/toscana/plugins/lifecycle/AbstractLifecycle.java

```diff
@@ -13,8 +18,17 @@
 public abstract class AbstractLifecycle implements TransformationLifecycle {
 
     /**
-     The transformation specific logger that can be used to log to the transformations Log 
-     Object
+     Declare transformation content specific directories name and paths
+     */
+    public static final String OUTPUT_DIR = "output/";
```

Comment:
Not quite sure about this code block. Any recommendations @hnicke @c-mueller?

Reply:
I'd remove the trailing slash; rest is fine.